exercises/space-age/src/example.h

```diff
@@ -0,0 +1,25 @@
+#ifndef EXAMPLE_H
+#define EXAMPLE_H
```

Comment:
Examples for our other exercises use the actual module name for the include guard. Can you change this to `SPACE_AGE_H`?